src/Tests/Microsoft.NET.Sdk.Razor.Tool.Tests/ServerCommandTest.cs

```diff
@@ -61,49 +61,29 @@ public void WritePidFile_WorksAsExpected()
         public void GetPidFilePath_ReturnsCorrectDefaultPath()
         {
             // Arrange
-            var expectedPath = Path.Combine("homeDir", ".dotnet", "pids", "build");
-            var server = GetServerCommand();
+            var expectedPath = Path.Combine(".dotnet", "pids", "build");
 
             // Act
-            var directoryPath = server.GetPidFilePath(getEnvironmentVariable: env =>
-            {
-                if (env == "DOTNET_BUILD_PIDFILE_DIRECTORY")
-                {
-                    return null;
-                }
-
-                return "homeDir";
-            });
+            var directoryPath = ServerCommand.GetPidFilePath();
 
             // Assert
-            Assert.Equal(expectedPath, directoryPath);
+            Assert.EndsWith(expectedPath, directoryPath);
         }
 
         [Fact]
         public void GetPidFilePath_UsesEnvironmentVariablePathIfSpecified()
         {
             // Arrange
             var expectedPath = "/Some/directory/path/";
-            var server = GetServerCommand();
+            Environment.SetEnvironmentVariable("DOTNET_BUILD_PIDFILE_DIRECTORY", expectedPath);
 
             // Act
-            var directoryPath = server.GetPidFilePath(getEnvironmentVariable: env => expectedPath);
+            var directoryPath = ServerCommand.GetPidFilePath();
 
             // Assert
             Assert.Equal(expectedPath, directoryPath);
-        }
 
-        [Fact]
-        public void GetPidFilePath_NullEnvironmentVariableValue_ReturnsNull()
-        {
-            // Arrange
-            var server = GetServerCommand();
-
-            // Act
-            var directoryPath = server.GetPidFilePath(getEnvironmentVariable: env => null);
-
-            // Assert
-            Assert.Null(directoryPath);
+            Environment.SetEnvironmentVariable("DOTNET_BUILD_PIDFILE_DIRECTORY", "");
```

Comment:
Nit - could you please wrap the rest of the test in try/finally and reset the env. variable in the finally block... so that it failing doesn't potentially cause cascade failures.